app/code/Transoft/Blog/view/frontend/templates/blogListTmpl.phtml

```diff
@@ -0,0 +1,24 @@
+<?php
+use Transoft\Blog\ViewModel\Bloglist;
+
+/** @var \Transoft\Blog\Block\Bloglist $block*/
+/** @var Bloglist $bloglistViewModel */
+
+$bloglistViewModel = $block->getData('bloglist_view_model');
+$data = $bloglistViewModel->getPostCollection()->getData();
```

Comment:
Please use item->get.... instead of arrays.

Reply:
fixed